examples/realtime/plot_compute_rt_average.py

```diff
@@ -55,6 +55,7 @@
         evoked = ev
     else:
         evoked += ev
+    evoked.pick_types(meg=True, eog=False)  # leave out the eog channel
```

Comment:
Which seems to be a bit inconsistent with constructing ``EvokedArray`` and using ``iter_evoked``...